bin/rtlcss.js

```diff
@@ -197,8 +205,9 @@ if (!shouldBreak) {
               }
             })
           } else {
-            // process only *.css
-            if (/\.(css)$/.test(file)) {
+            // process only *.css and ignore RTLed files
+            var patt = new RegExp('('+ext+')', 'gm');
```

Comment:
OK, so we need to add backslash for regexp characters; `var _ext = ext.replace('.', '\.');` ???